src/utils/mixins.js

```diff
@@ -18,3 +18,22 @@ export const sizeFormatter = {
     },
   },
 };
+
+/**
+ * Any component that needs form validation.
+ */
+const validator = {
+  $_veeValidate: {
+    validator: 'new',
+  },
+  methods: {
+    async validate() {
+      return this.$validator.validateAll();
+    },
+  },
+}
+
+export {
+  sizeFormatter, 
+  validator,
+}
```

Comment:
Please change editor settings so that they add final newline to files automatically on save to avoid these.